.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!--- Provide a general summary of the issue in the Title above -->
```

Comment:
Is this template necessary? Feature request template and bug report template is already there with this PR.

Reply:
Will remove it.